controllers/pre_backup.go

```diff
@@ -259,7 +257,7 @@ func prepareImportedClusters(ctx context.Context,
 				msaAddon := &addonv1alpha1.ManagedClusterAddOn{}
 				msaAddon.Name = msa_addon
 				msaAddon.Namespace = managedCluster.Name
-				msaAddon.Spec.InstallNamespace = installNamespace
+				// Not setting msaAddon.Spec.InstallNamespace - will leave default
```

Comment:
I think keep it empty is ok since there is a [default value](https://github.com/open-cluster-management-io/api/blob/2b2399b5f6e88e28a962aa82307ccc00bc492009/addon/v1alpha1/types_managedclusteraddon.go#L39): "open-cluster-management-agent-addon" (added from ACM 2.7)